react-common/styles/react-common-breakpoints.less

```diff
@@ -1,17 +1,22 @@
 @mobileBreakpoint: 320px;
+@shareModalBreakpoint: 440px;
```

Comment:
I don't like using non-standard breakpoints like this. can we look into altering the design to works with the existing breakpoints? if not, lets put this breakpoint in the share's less files and not the common one.

Reply:
Yeah, I'll remove these for now, and hopefully we can get a solution for everything working together for smaller screens in a later change.

Reply:
I do need to make a new rule, though, because there needs to be an in-between for smaller tablet sizes and the like. From playing with it, I think that the modal should switch flex-directions at smaller screens than we currently have it.